runConfigurations/Cromwell_server.run.xml

```diff
@@ -8,6 +8,8 @@
       <env name="CROMWELL_BUILD_RESOURCES_DIRECTORY" value="target/ci/resources" />
       <env name="CROMWELL_BUILD_PAPI_JSON_FILE" value="target/ci/resources/cromwell-centaur-service-account.json" />
       <env name="CROMWELL_BUILD_CENTAUR_READ_LINES_LIMIT" value="128000" />
+      <env name="CROMWELL_BUILD_PAPI_DOCKER_IMAGE_DRS" value="broadinstitute/cromwell-drs-localizer:develop" />
+      <env name="CROMWELL_BUILD_CENTAUR_256_BITS_KEY" value="AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=" />
```

Comment:
Yes exactly. I'm probably going to delete the first line because that's something a developer working on the DRS localizer should explicitly specify to avoid difficult-to-debug surprises.